language.txt

```diff
@@ -76,6 +76,31 @@ Unicode code point."
 `AbCdEf`
 
 
+# Bit string literals.
+
+# A bit string literal is a hex literal beginning with 'b'. Only 0 and 1 may
+# appear inside. Bit string literals are encoded like a BIT STRING. (Big-endian
+# bit order, prefixed with a "number of padding bits" octet.)
+
+# This encodes as `0055`.
+b`10101010`
+
+# This encodes as `0450`.
+b`1010`
+
+# Bit string literals may also include a single | character, which marks the
+# beginning of explicit padding bytes. BER permits any bit sequence for the
+# padding bytes. However, it is an error for explicit padding to cross the octet
```

Comment:
s/the/an/ since there are many octet boundaries. Also looks like I used "byte" in the most of the rest of the document, so feel free to use the less silly term. :-)